include/networkit/auxiliary/BucketPQ.hpp

```diff
@@ -99,6 +108,16 @@ class BucketPQ: public PrioQueue<int64_t, index> {
      */
     uint64_t size() const override;
 
+    /**
+     * @return Whether or not the PQ is empty.
+     */
+    bool empty() const noexcept override;
+
+    /**
+     * @return Whether or not the PQ contains the given element.
+     */
+    bool contains(const index &value) const override;
```

Comment:
Take `index` by value. `index` is a small integer type that is cheap to copy.

Reply:
I agree but this is already defined in `PrioQueue` which takes a const ref. The same is done in other methods like `remove` because `PrioQueue` can also store object that are expensive to copy.